tests/test_wgpu_occlusion_query.py

```diff
@@ -0,0 +1,172 @@
+"""
+Test occlusion queries.
+"""
+
+import numpy as np
+import sys
+
+import wgpu
+from pytest import skip
+from testutils import run_tests, get_default_device
+from testutils import can_use_wgpu_lib, is_ci
+from wgpu import flags
+
+if not can_use_wgpu_lib:
+    skip("Skipping tests that need the wgpu lib", allow_module_level=True)
+elif is_ci and sys.platform == "win32":
```

Comment:
Is it possible to make this conditional on DirectX being used?

Reply:
At some point we should get back to these and see if they work now on CI. Also see #459